app/javascript/src/components/Dashboard/Notes/SideMenu.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+
+import { Search, Plus, Settings } from "neetoicons";
+import { Typography } from "neetoui";
+import { MenuBar } from "neetoui/layouts";
+
+const SideMenu = () => {
```

Comment:
Why is there a `NotesMenu` as well as a `SideMenu` within the `Notes` folder?

Reply:
I forgot to delete the NotesMenu file in this branch.

Reply:
Forgetting things is in the nature of human beings. That's why we have a step called `self-review` to overcome the mistakes we might've made cause we had forgotten. The important step is checking what has been added/changed/removed from the `files changed` tab in Github. Read: https://handbook.neetokb.com/articles/git-and-github-essentials#self-review-process

Reply:
Would always do review the codes on github very carefully from now on.